thumbor/engines/extensions/exif_orientation_editor.py

```diff
@@ -0,0 +1,66 @@
+import os
+import struct
+from io import BytesIO
+
+PREFIXES = [
+    b"MM\x00\x2A",  # Valid TIFF header with big-endian byte order
+    b"II\x2A\x00",  # Valid TIFF header with little-endian byte order
+    b"MM\x2A\x00",  # Invalid TIFF header, assume big-endian
+    b"II\x00\x2A",  # Invalid TIFF header, assume little-endian
+]
+
+EXIF_HEADER = b"Exif\x00\x00"
+
+
+class ExifOrientationEditor():
+    _endian = "<"
+    _offset = None
+
+    def __init__(self, data):
+        if data[:6] != EXIF_HEADER:
+            raise SyntaxError("not a TIFF file (header %r not valid)" % data[:4])
+
+        self.fp = BytesIO(data[6:])
+        ifh = self.fp.read(8)
```

Comment:
What does `ifh` mean?

Reply:
It might be helpful to comment why this is only reading out 8 bytes. May be self documenting if `ifh` had a more descriptive name?